commit 803164b3e8999d7053cfa61e6c226d5cc8d026f2 (refs/changes/77/3377/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-02-15T16:31:15-08:00 (1 year, 8 months ago)
    
    TRITON-151 node-sdc-clients vmapi tests are failing "wait for add nics using networks job ... Error: Timeout after 90 seconds"

diff --git a/test/vmapi.test.js b/test/vmapi.test.js
index 22faa7b..89816ce 100644
--- a/test/vmapi.test.js
+++ b/test/vmapi.test.js
@@ -284,8 +284,8 @@ test('vmapi', function (tt) {
 
             napi.listNetworks({ name: 'external' }, function (err2, nets2) {
                 t.ifError(err2);
-                t.ok(nets1);
-                EXTERNAL_NETWORK = nets1[0].uuid;
+                t.ok(nets2);
+                EXTERNAL_NETWORK = nets2[0].uuid;
                 t.end();
             });
         });
@@ -816,6 +816,7 @@ test('vmapi', function (tt) {
             t.ifError(err);
             t.ok(job);
             JOB_UUID = job.job_uuid;
+            t.ok(JOB_UUID, 'JOB_UUID is ' + JOB_UUID);
             t.end();
         });
     });
